datamap/mv/core/src/test/scala/org/apache/carbondata/mv/rewrite/MVCreateTestCase.scala

```diff
@@ -829,16 +829,14 @@ class MVCreateTestCase extends QueryTest with BeforeAndAfterAll {
   test("jira carbondata-2533") {
 
     sql("drop datamap if exists MV_exp")
-    intercept[UnsupportedOperationException] {
-      sql(
-        "create datamap MV_exp using 'mv' as select sum(case when deptno=11 and (utilization=92) then salary else 0 end) as t from fact_table1 group by empname")
+    sql(
+      "create datamap MV_exp using 'mv' as select sum(case when deptno=11 and (utilization=92) then salary else 0 end) as t from fact_table1 group by empname")
 
-      sql("rebuild datamap MV_exp")
-      val frame = sql(
-        "select sum(case when deptno=11 and (utilization=92) then salary else 0 end) as t from fact_table1 group by empname")
-      val analyzed = frame.queryExecution.analyzed
-      assert(verifyMVDataMap(analyzed, "MV_exp"))
-    }
+    sql("rebuild datamap MV_exp")
```

Comment:
give test case a proper name, and better to mention jira ID also, as it will easy for future reference

Reply:
done